plugins/template.md

```diff
@@ -329,9 +329,7 @@ This is the contents your backend page should return if you specify a URL in the
 
 ### Making Templates
```

Comment:
Should these headings become something like "Making Template Files" and "Making Inline Templates"? Idk what language we use elsewhere but these mean basically nothing to me

Reply:
And reference the content/url thing or whatever it is?

Reply:
fixed